pkgs/development/tools/misc/n98-magerun/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, stdenv, fetchurl, fetchFromGitHub, php, which, makeWrapper, bash, coreutils, ncurses, formats }:
+
+stdenv.mkDerivation rec {
+  pname = "n98-magerun";
+  version = "2.3.0";
+
+  src = fetchFromGitHub {
+      owner = "netz98";
+      repo = "n98-magerun";
+      rev = "${version}";
+      forceFetchGit = true;
+      sha256 = "/RffdYgl2cs8mlq4vHtzUZ6j0viV8Ot/cB/cB1dstFM=";
+    };
+
+  nativeBuildInputs = [
+    makeWrapper
+    php
+    php.packages.composer
+  ];
+
+  buildPhase = ''
+    runHook preBuild
+    composer install --dev
+    php -f vendor/bin/phing -dphar.readonly=0 -- dist_clean
```

Comment:
note for reviewers: I couldn't use php.packages.phing because it wouldn't have allowed me to pass the "phar.readonly=0" parameter which is needed (by php security) in order to write to a phar.

At least I couldn't find a way to do it.